cli/cmd/root.go

```diff
@@ -65,12 +66,13 @@ This will prompt you for your Lacework account and a set of API access keys.`,
 					return nil
 				}
 
-				if err := cli.NewClient(); err != nil {
-					return err
+				err := cli.NewClient()
+				if err != nil && strings.Contains(err.Error(), "Invalid Account") {
```

Comment:
we are slowing the error from `NewClient()` if it doesn't contain `Invalid Account`